server/cromwell.py

```diff
@@ -171,11 +171,12 @@ async def cromwell(request):
             job = batch.new_job(name='driver')
             job = prepare_git_job(
                 job=job,
+                dataset=dataset,
                 access_level=access_level,
+                output_dir=output_dir,
```

Comment:
```
            if access_level == 'test':
                workflow_output_dir = f'gs://cpg-{dataset}-test/{output_dir}'
            else:
                workflow_output_dir = f'gs://cpg-{dataset}-main/{output_dir}'
```

Maybe putting it next to: https://github.com/populationgenomics/analysis-runner/blob/c9450a3783a01c6eb43c21ea1290a456f2e51915/server/cromwell.py#L117-L120

Reply:
Thanks, almost had this identical change, but I'll update it with your variable name :).